lib/theme_check/language_server/server.rb

```diff
@@ -52,8 +52,19 @@ def send_response(response)
         response_body = JSON.dump(response)
         log(JSON.pretty_generate(response)) if $DEBUG
 
-        @out.write("Content-Length: #{response_body.bytesize}\r\n")
-        @out.write("\r\n")
+        # Because programming is fun,
+        #
+        # Ruby on Windows turns \n into \r\n. Which means that \r\n
+        # gets turned into \r\r\n. Which means that the protocol
+        # breaks on windows unless we turn STDOUT into binary mode and
+        # set the encoding manually (yuk!) or we do this little hack
+        # here and put \n which gets transformed into \r\n on windows
+        # only...
+        #
+        # Hours wasted: 8.
+        eol = Gem.win_platform? ? "\n" : "\r\n"
+        @out.write("Content-Length: #{response_body.bytesize}#{eol}")
```

Comment:
for clarity, I'd suggest breaking out the line termination from the http header name/value pair. also, any concern that the bytesize will be null? If so, I'd want to add a safety here. (I'm guessing not since the response_body is fully buffered?